modules/tensor_mechanics/src/materials/ADGeneralizedRadialReturnStressUpdate.C

```diff
@@ -38,6 +38,8 @@ ADGeneralizedRadialReturnStressUpdate::validParams()
   params.addRequiredParam<std::string>(
       "inelastic_strain_rate_name",
       "Name of the material property that stores the inelastic strain rate");
+  params.addParam<RealVectorValue>("rotation_angles",
+                                   "Provide the rotation angle for transformation matrix");
```

Comment:
If I understand well, `rotation_angles` here refers to the ZYX Euler angle sequence. It may be good to put this in the description if that's the case.

Reply:
Done.